### ISSUE TITLE
Read more is not localized

### ISSUE THREAD

**Issue by Mte90** [yithemes/yith-proteo#28]
As you can see on https://github.com/yithemes/yith-proteo/blob/e8149cf57e41306af164ee900f66d41beb83776c/inc/template-functions.php#L261
 the string doens't have the textdomain so is not translated also if the string localization is in the po file.

**Comment by francgrasso** [comment]
Hi there,
the text can be changed from the customizer: https://prnt.sc/1rllnk4

**Comment by Mte90** [comment]
Yes but in a multilanguage website I need that is localized, in the po file there is the string but this code ignore the case that the parameter in th customizer is empty and use that one.

**Comment by Mte90** [comment]
![immagine](https://user-images.githubusercontent.com/403283/132654375-18f894fd-831c-475e-8292-85503b40da61.png)
Seems also as how the file names are structured in the theme they are not loaded by wordpress as in languages the file has the theme prefix but natively is not looking for them with that name

**Comment by Mte90** [comment]
```
function yith_proteo_modify_read_more_link() {
    $string = get_theme_mod( 'yith_proteo_blog_read_more_text' );
    if ( empty( $string ) ) {
        $string = __( 'Read more  &#10230;', 'yith-proteo' );
    }
	return '<a class="more-link" href="' . get_permalink() . '">' . esc_html( $string ) . '</a>';
}

add_filter( 'the_content_more_link', 'yith_proteo_modify_read_more_link' );
```

I changed the code in this way

**Comment by Mte90** [comment]
ping

**Comment by francgrasso** [comment]
Hi there, fixed in commit b4c080e
